airflow/cli/cli_parser.py

```diff
@@ -1359,8 +1359,20 @@ class GroupCommand(NamedTuple):
     ActionCommand(
         name='upgrade',
         help="Upgrade the metadata database to latest version",
+        description=(
+            "Upgrade the schema of the metadata database. "
+            "To print but not execute commands, use option `--sql-only`. "
```

Comment:
Should we name this `--sql`? closer to alembic offline migration command

Reply:
We probably have 3 options
1. `--sql-only`
2. `--show-sql-only`
3. `--sql`

Reply:
> Should we name this --sql? closer to alembic offline migration command

I don't think so.  As a user, think of UX.  they don't use alembic, in general. `sql` alone does not tell you enough about what it's doing.  the "only" signifies that it's "only" sql and not also something else  -- namely running the migrations.

i'd be ok with `show-only` or `show-sql-only` if that's what the group prefers but i don't think `sql` alone is ok.